myuw/dao/class_website.py

```diff
@@ -19,9 +20,15 @@ def _fetch_url(method, url):
         logger.error("_get_html_from_url(%s)==>%s" % (url, ex))
         return None
 
-    pool = get_con_pool("%s://%s" % (p.scheme, p.netloc), socket_timeout=2)
-    response = get_live_url(
-        pool, method, p.hostname, url, {'ACCEPT': 'text/html'})
+    headers = {'ACCEPT': 'text/html'}
+    if p.netloc == 'localhost':
```

Comment:
the thought was to minimize config and figured since "localhost" has a pretty specific meaning so it's clear what what to expect when you see it in the mock resource.

if it's in config, what's the default?  are you suggesting mocked resources with links to mock websites get changed too?